docs/ecosystem.md

```diff
@@ -52,36 +44,36 @@ and other C modules, encapsulating the pointer with a name and optional destruct
 manage resources and ensure safe access,
 see [here](https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html) for details.
 
-Narwhals supports exporting a DataFrame via the Arrow PyCapsule Interface.
+Narwhals supports exporting a DataFrame via the Arrow PyCapsule Interface. See
+[Universal dataframe support with the Arrow PyCapsule Interface + Narwhals](https://labs.quansight.org/blog/narwhals-pycapsule)
+for how you can use them together.
 
 ### Ibis
 
 Pitched as "The portable Dataframe library", Ibis provides a Pythonic frontend
 to various SQL (as well as Polars LazyFrame) engines. Some differences with Narwhals are:
 
-* Narwhals' main use case is for library maintainers wanting to support
-  different dataframe libraries without depending on any whilst keeping
-  things as lightweight as possible. Ibis is more targeted at end users
-  and aims to be thought of as a Dataframe library akin to
-  pandas / Polars / etc.
-* Narwhals allows you to write a "Dataframe X in, Dataframe X out" function.
+* Narwhals allows you to write "Dataframe X in, Dataframe X out" functions.
   Ibis allows materialising to pandas, Polars (eager), and PyArrow, but has
   no way to get back to the input type exactly (e.g. there's no way to
-  start with a Polars LazyFrame and get back a Polars LazyFrame)
+  start with a Polars LazyFrame and get back a Polars LazyFrame with all
+  untouched columns unmodified).
```

Comment:
I am not the biggest fan of the phrasing: "with all untouched columns unmodified"
Maybe something along the lines of "without modifying the initial object"

Reply:
i don't think either modifies the original object, i just wanted to get across that in Narwhals you can perfectly round-trip

have rephrased, thanks!